src/ml_tooling/plots/target_feature_distribution.py

```diff
@@ -0,0 +1,72 @@
+from matplotlib import pyplot as plt
+from matplotlib.axes import Axes
+import pandas as pd
+import numpy as np
+
+from ml_tooling.utils import DataType
+from scipy import stats
+
+
+def target_feature_distribution(
+    data: pd.DataFrame,
+    targets: DataType,
+    features: DataType,
+    title: str = None,
+    labels: list = None,
+    median: bool = False,
```

Comment:
This should probably be a string selecting aggregation method - otherwise we would end up with multiple flags if we want to add more aggregation methods and would have to check for each combination of them